exec/Chainweb/Backfill.hs

```diff
@@ -72,6 +87,74 @@ backfill e delay = do
         hs -> traverse_ (writeBlock e pool count) hs
       delayFunc
 
+backfillEvents :: Env -> BackfillArgs -> IO ()
+backfillEvents e args = do
+  cutBS <- queryCut e
+  let curHeight = fromIntegral $ cutMaxHeight cutBS
+      cids = atBlockHeight curHeight allCids
+  counter <- newIORef 0
+
+  missingTxs <- countTxMissingEvents pool
+
+  -- assume a default value suitable for mainnet... the value 11138000 is the
+  -- height at which events were activated in chainweb-node
+  missingCoinbase <- countCoinbaseTxMissingEvents pool eventsActivationHeight
+  if M.null missingTxs && M.null missingCoinbase
+    then do
+      printf "[INFO] There are no events to backfill on any of the %d chains!" (length cids)
+      exitSuccess
+    else do
+      let numTxs = foldl' (+) 0 missingTxs
+          numCoinbase = foldl' (\s h -> s + (h - fromIntegral eventsActivationHeight)) 0 missingCoinbase
+
+      printf "[INFO] Beginning event backfill of %d txs on %d chains.\n" numTxs (M.size missingTxs)
+      printf "[INFO] Also beginning event backfill (of coinbase transactions) of %d txs on %d chains.\n" numCoinbase (M.size missingCoinbase)
+      let strat = case delay of
+                    Nothing -> Par'
+                    Just _ -> Seq
+      race_ (progress counter $ fromIntegral (numTxs + numCoinbase))
+        $ traverseConcurrently_ strat (f missingCoinbase counter) cids
+
+  where
+    delay = _backfillArgs_delayMicros args
+    coinbaseDelay = _backfillArgs_coinbaseDelayMicros args
+    pool = _env_dbConnPool e
+    allCids = _env_chainsAtHeight e
+    delayFunc =
+      case delay of
+        Nothing -> pure ()
+        Just d -> threadDelay d
+    eventsActivationHeight = fromMaybe 1138000 $ _backfillArgs_eventsActivationHeight args
```

Comment:
If we're going to put a hard-coded default in here, I think we should case on the chainwebVersion and use the correct value depending on whether it's mainnet01 or testnet04.